.github/workflows/cd.yml

```diff
@@ -0,0 +1,28 @@
+name: cd
+
+on:
+  push:
+    branches: [master]
+
+  workflow_dispatch:
+    inputs:
+      run_service_name:
+        description: 'Cloud Run service name'
+        required: true
+        default: 'github-app-notify-twitter'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: google-github-actions/deploy-cloudrun@v0.9.0
+        with:
+          service: ${{ secrets.GCP_CLOUDRUN_SERVICE_NAME }}
+          region: ${{ secrets.GCP_CLOUDRUN_SERVICE_REGION }}
+          credentials: ${{ secrets.GCP_SA_KEY }}
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          env_vars: TWITTER_API_KEY=${{ secrets.TWITTER_API_KEY }},TWITTER_API_SECRET=${{ secrets.TWITTER_API_SECRET }},TWITTER_ACCESS_TOKEN=${{ secrets.TWITTER_ACCESS_TOKEN }},TWITTER_ACCESS_TOKEN_SECRET=${{ secrets.TWITTER_ACCESS_TOKEN_SECRET }}
```

Comment:
for readability, let's break them into multiple lines. this website helps figuring out which syntax to use exactly https://yaml-multiline.info/